packages/taiko-client/driver/txlist_fetcher/blob.go

```diff
@@ -92,12 +93,28 @@ func (d *BlobFetcher) FetchPacaya(
 		return nil, pkg.ErrBlobUnused
 	}
 
+	var blockNum uint64
+
+	if meta.GetBlobCreatedIn().Int64() == 0 {
```

Comment:
Will the log code be removed later?

Reply:
we can remove all log code now tbh, I only needed it to help figure out what was wrong.